core/templates/dev/head/pages/exploration_player/conversation_skin_directive.html

```diff
@@ -144,6 +148,16 @@
   }
 
   /* All classes below should start with .conversation-skin */
+  .conversation-skin-back-to-collection-container {
+    margin-top: 25px;
+    margin-bottom: 92px;
```

Comment:
This 92px was chosen to 1) Match margin style between the text "Learned something new..." and the element above it, and 2) be above the height of the footer and it's content (which is margin: 30px, element height 42px)

This yields 30px + 42px (Even with the footer) + 20px (consistent with the margin I mentioned above)

Please see screenshots for more information.

Margin I tried to match (20px):
![margin1](https://user-images.githubusercontent.com/6820349/27164703-ac239b6a-5154-11e7-953f-dc9ae38fab97.png)

Margin of the link (20px + the margin and height of the folder, more detailed calculation above)
![margin2](https://user-images.githubusercontent.com/6820349/27164702-ac22afe8-5154-11e7-90ed-b527c147bf62.png)